app/iot/devices.py

```diff
@@ -1,3 +1,4 @@
+import asyncio
 import time
```

Comment:
It seems that the 'time' module is imported but never used in this file. It's a good practice to remove unused imports to keep the code clean and easy to read.